src/fractal-contract.cpp

```diff
@@ -284,7 +284,8 @@ void fractal_contract::submitcons(const uint64_t& groupnr, const std::vector<nam
 
     check(is_account(submitter), "Submitter's account does not exist.");
 
-    for (int i = 0; i < rankings.size(); i++) {
+     for (size_t i = 0; i < rankings.size(); i++)
```

Comment:
For some reason I see a bunch of compiler warnings on this line
`/root/eos/src/fractal-contract.cpp:287:28: warning: treating Unicode character as whitespace [-Wunicode-whitespace]`

Reply:
I pushed a fix for this